authors/apps/authentication/tests/test_verify_user.py

```diff
@@ -0,0 +1,26 @@
+from authors.apps.authentication.tests.basetests import BaseTest
+from rest_framework import status
+
+class TestUserVerification(BaseTest):
+    """
+    This class handles the testing of User Verification
+    """
+
+    def test_successfull_verification(self):
+        token = self.generate_jwt_token('pablo@escobar.com', 'Escobar')
+        url = "/api/users/activate/" + token
```

Comment:
you could have the URL patterns in the base file

Reply:
Okay will work on this with a bug fix